distribution/docs/src/main/jdocs/content/_plugins/identification-plugin-contents.adoc

```diff
@@ -0,0 +1,17 @@
+:type: plugin
+:status: published
+:title: Identification Plugin
+:link: _identification_plugin
+:plugintypes: preingest,postingest
+:summary: Manages IDs on registry metacards.
+
+The Identification Plugin assigns IDs to registry metacards and adds/updates IDs on create and update.
```

Comment:
Don't know if it is worthwhile mentioning that this plugin also is responsible for toggling the registry tags. `For incoming remote registry ingest operations, metacards tagged as 'registry' will be converted to 'registry-remote'` That might be too implementation specific.